test/end-to-end/test_tar_export.sh

```diff
@@ -34,14 +34,15 @@ fi
 # Remove tarball if already present
 rm -f ./*.tar.gz
 
-/src/target/debug/hab pkg export tar "$pkg_ident" 
+hab pkg export tar "$pkg_ident" --base-pkgs-channel=DEV
```

Comment:
Should this channel be configurable?

Reply:
nope but it should be pulling from the ENV